lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -508,7 +508,7 @@ class MockTimers {
       );
     }
 
-    validateArray(timers, 'timers');
+    validateStringArray(timers, 'timers');
```

Comment:
I assume this is semver-major, correct? If so, can you please take it as its own PR?

Reply:
@aduh95 the `MockTimers` is currently experimental, I don't think we have to mark this as https://github.com/nodejs/node/labels/semver-major
https://github.com/nodejs/node/blob/c86e700b3ee5efc4f5b3ed878484b5d50f34eec9/doc/api/test.md#L1560-L1568

Reply:
Actually I'm not sure this PR is `semver-major` or not. How about checking jenkins-ci's result according to below collaborator guide?
https://github.com/nodejs/node/blob/7bf29b52b3c54a071b814ccf77cfaf275279136f/doc/contributing/collaborator-guide.md?plain=1#L901-L908